packages/rspack/src/builtin-plugin/index.ts

```diff
@@ -279,6 +279,31 @@ export function deprecated_resolveBuiltins(
 		noEmitAssets = true;
 	}
 
+	if (options.experiments.rspackFuture?.disableTransformByDefault) {
+		(
+			[
+				"react",
+				"pluginImport",
+				"decorator",
+				"presetEnv",
+				"emotion",
+				"relay"
+			] as const
+		).forEach(key => {
+			if (builtins[key]) {
+				deprecatedWarn(
+					`'builtins.${key} = ${JSON.stringify(
+						builtins[key]
+					)}' has been deprecated, please migrate to ${termlink(
+						"builtin:swc-loader options",
+						"https://www.rspack.dev/guide/loader.html#builtinswc-loader"
+					)}`,
+					true
```

Comment:
Would you remove the second parameter? Adding `true` here will result in this cannot be turned off by env.

Reply:
It may be inappropriate to use deprecatedWarn, because it doesn't worked when disableTransformByDefault is turned on, so I don't think it can be turned off by env.